package_manifest.yaml

```diff
@@ -151,6 +151,10 @@ foreman_scl_packages:
     rubygem-record_tag_helper: {}
     rubygem-representable: {}
     rubygem-responders: {}
+    rubygem-rest-client:
+      releasers:
+        - koji-foreman
+        - koji-foreman-plugins
```

Comment:
This shouldnt need to exist in plugins if foreman repository has it.

Reply:
foreman: SCL
plugins: nonSCL
:)

Reply:
Helps if I read all your comments instead of just the first.